### DISCUSSION TITLE
How do I prevent automatic building?

### DISCUSSION THREAD

**Discussion by ntreeby** [vscode-abl/vscode-abl#29] [Q&A] [answered]
In PDSOE, I can tell the project not to build automatically, and that's it - it never tries to build.
How would I achieve the same with vscode-abl?

I have an environment config project, which has many PowerShell scripts, JSON config files, tokenized templates, .war files, XML, etc.  And, a bunch of .p's that are meant for data loading/transformation, and aren't tied to any particular database - we have many common "framework" tables in all 3 of our primary application databases.  These .p's are intended to be run uncompiled, against any database, either manually on a developer's machine, or on a TFS agent on our build/deployment servers, which also have dev licenses.

So - I don't really want or need to compile these .p's.  But, I still want syntax highlighting and all the other useful features of vscode-abl.

**Comment by gquerret** [comment, marked as answer]
If your project is only made of files that don't compile, you can either delete the openedge-project.json (syntax highlight will still work, but not a lot more), or have no 'source' directory in the buildPath attribute (nothing will be compiled), or use the 'excludes' attribute of source directories.
